vtr_flow/arch/titan/README.rst

```diff
@@ -24,46 +26,72 @@ CAD evaluation).
 
 Directory Structure
 --------------------------------------------------
+
 stratixiv_arch.timing.xml:
-    A capture of Altera's Stratix IV FPGA architecture. It makes some 
-    relatively minor architectural approximations to be compatible with
-    VPR.  It includes a timing model which has been calibrated to the
-    Stratix IV timing model included in Altera's Quartus II CAD tools.
+    An enhanced capture of Altera's Stratix IV FPGA architecture. It makes some 
+    relatively minor architectural approximations to be compatible with VPR. It
+    includes a timing model which has been calibrated to the Stratix IV timing
+    model included in Altera's Quartus II CAD tools.
+
```

Comment:
Compared to the original (legacy) capture of Stratix IV, this architecture better captures the routing architecture by modeling the hierarchical connections between the L16 and L4 networks, and the custom switch patterns between them.  The L4 wires have 12-input muxes driving them, and the L16 wires have 40:1 muxes driving them, roughly matching the Stratix IV mux sizes from the information available.

This is the architecture used to generate results in the VTR 8.0 TRETS paper, and corresponds to architecture E in Table 3 in that paper.

Reply:
(Please add comment above to text).

Reply:
Thank you for reviewing. I've added these comments as suggested.